packages/ubuntu_desktop_installer/lib/pages/who_are_you/who_are_you_page.dart

```diff
@@ -0,0 +1,368 @@
+import 'package:encrypt/encrypt.dart' as encrypt;
+import 'package:flutter/material.dart';
+import 'package:form_field_validator/form_field_validator.dart';
+import 'package:provider/provider.dart';
+import 'package:subiquity_client/subiquity_client.dart';
+import 'package:yaru/yaru.dart' as yaru;
+
+import '../../routes.dart';
+import '../../widgets/localized_view.dart';
+import '../../widgets/validated_input.dart';
+import '../wizard_page.dart';
+import 'who_are_you_model.dart';
+
+/// The installer page for setting up the user data.
+///
+/// It uses [WizardPage] and [WizardAction] to create an installer page.
+class WhoAreYouPage extends StatefulWidget {
+  /// Creates a the installer page for setting up the user data.
+  const WhoAreYouPage({Key? key}) : super(key: key);
+
+  ///
+  static Widget create(BuildContext context) {
+    return ChangeNotifierProvider(
```

Comment:
`final client = Provider.of<SubiquityClient>(context, listen: false);` and pass it to `WhoAreYouModel` constructor so it will be able to call the client in a testable way